### ISSUE TITLE
Usage of external library 'shell32'

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#935]
````
&scoped-define SHELL32.DLL "shell32"

PROCEDURE ShellExecuteW EXTERNAL {&SHELL32.DLL}.
   DEFINE INPUT          PARAMETER hwnd             AS {&HWND}.
   DEFINE INPUT          PARAMETER lpOperation      AS CHARACTER.
   DEFINE INPUT          PARAMETER lpFile           AS CHARACTER.
   DEFINE INPUT          PARAMETER lpParameters     AS CHARACTER.
   DEFINE INPUT          PARAMETER lpDirectory      AS CHARACTER.

END PROCEDURE.
````
Is flagged, even though the rule description states:

> By default, the standard Microsoft Windows libraries are considered safe, as well as the C library on Unix / Linux

Is this a result of the external not having an extension? (this code has been working fine since the previous millennium).

**Comment by gquerret** [comment]
That's definitely a FP. I'll double-check the exclusion rules.

**Comment by stefandrissen** [comment]
Another, which is using a full dll name, is also being flagged:
````
   PROCEDURE gethostname EXTERNAL "wsock32.dll".
      DEFINE       OUTPUT  PARAMETER p-Hostname      AS CHARACTER.
      DEFINE INPUT         PARAMETER p-Length        AS LONG.
      DEFINE       RETURN  PARAMETER p-Return        AS LONG.
   END PROCEDURE.
````

**Comment by gquerret** [comment]
The default value now includes a longer list of libraries. Note that you can modify the list directly in the quality profile.
